sound/soc/sof/ipc4-topology.c

```diff
@@ -0,0 +1,836 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
+//
+// Copyright(c) 2021 Intel Corporation. All rights reserved.
+//
+//
+
+#include <linux/bits.h>
+#include <linux/device.h>
+#include <linux/errno.h>
+#include <linux/firmware.h>
+#include <linux/workqueue.h>
+#include <sound/pcm_params.h>
+#include <sound/sof/topology.h>
+#include <uapi/sound/sof/tokens.h>
+#include "ops.h"
+#include "sof-audio.h"
+#include "sof-priv.h"
+#include "ipc4.h"
+#include "ipc4-topology.h"
+#include "topology-common.h"
+
+static const struct sof_topology_token ipc4_sched_tokens[] = {
+	{SOF_TKN_SCHED_LP_MODE, SND_SOC_TPLG_TUPLE_TYPE_WORD, get_token_u32,
+		offsetof(struct sof_ipc4_pipeline, lp_mode), 0}
```

Comment:
why does this need to be a seaprate array? It doesnt clash with any existing scheduler tokens for IPC3. So why not just include it in there?